audb/core/load.py

```diff
@@ -169,18 +169,15 @@ def _files_duration(
         files: typing.Sequence[str],
         format: typing.Optional[str],
 ):
-
-    def fix_file(file: str):
-        file = os.path.join(db.root, file)
-        if format is not None:
-            file = audeer.replace_file_extension(file, format)
-        return file
-
     field = define.DEPEND_FIELD_NAMES[define.DependField.DURATION]
     durs = deps._df.loc[files][field]
     durs = durs[durs > 0]
     durs = pd.to_timedelta(durs, unit='s')
-    durs.index = durs.index.map(fix_file)
+    if format is not None:
+        cur_ext = re.compile(r'\.[a-zA-Z0-9]+$')  # match file extension
+        new_ext = f'.{format}'
+        durs.index = durs.index.str.replace(cur_ext, new_ext, regex=True)
+    durs.index = db.root + os.path.sep + durs.index
```

Comment:
Since we need to expand the file path in other places, too (e.g. `auglib`) should we make it a utility function in `audformat`?

Reply:
I created https://github.com/audeering/audformat/pull/119

Reply:
And I https://github.com/audeering/audformat/pull/120, seems I'm too late.

Reply:
Sorry, didn't know you where also working on this.

Reply:
Not your fault, I think we started at the same time. My original goal was to continue work on `auglib`, but there you also asked for the utility function in `audformat` and then I decided to try it ;)

Reply:
I switched now to the new `audformat` release and use the corresponding helper functions.
I also made sure that the speed improvement is still as reported.